cellxgene_schema_cli/cellxgene_schema/write_labels.py

```diff
@@ -327,6 +330,17 @@ def _remove_categories_with_zero_values(self):
     def _build_schema_reference_url(self, schema_version: str):
         return f"{SCHEMA_REFERENCE_BASE_URL}/{schema_version}/{SCHEMA_REFERENCE_FILE_NAME}"
 
+    def _get_observation_joinid_column(self):
+        """
+        Set column with unique join ID for each row in obs dataframe.
+        """
+        return (
+            self.adata.obs.index.to_series()
```

Comment:
This command is mouthful and it also being use in the test fixtures. Is there a way to reuse this to avoid the test fixture implementation becoming out of sync with this version.